docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
```

Comment:
isn't it "host defin**er** cr? I thought the host definition is a separate CR

Reply:
you are right

Reply:
then please sync with @rivkap about this

Reply:
updated

Reply:
I still see:
>Installing the `HostDefinition` custom resource

in this PR diff

Reply:
updated - waiting to check in my changes until more information is received